src/templates/index.tsx

```diff
@@ -56,7 +58,7 @@ export const getPath = (data: any) => {
 const Index = ({ data }: { data: any }) => {
   const { document } = data;
   const { streamOutput } = document;
-  const { name, address, openTime, hours, mainPhone, _site, geocodedCoordinate, services, photoGallery } = streamOutput;
+  const { name, address, openTime, hours, mainPhone, _site, geocodedCoordinate, services, photoGallery, locale, id } = streamOutput;
```

Comment:
the Proxy logger picks up `id` here, even though it isn't used anywhere. Not a huge issue but maybe worth mentioning, we should probably avoid assigning variables we don't use anyways.

Reply:
gtk, definitely in favor of a go style linter that complains about unused variables.